File: tests/test_data/ChatGPT/v1/Transcribed/Conversor.py
def convertir_a_fahrenheit(celsius):
    return (celsius * 9/5) + 32

def main():
    try:
        celsius = float(input("Ingrese la temperatura en Celsius: "))
        fahrenheit = convertir_a_fahrenheit(celsius)
        print(f"Equivalente en Fahrenheit: {fahrenheit}")
    except ValueError:
        print("La entrada no es un número")

if __name__ == "__main__":
    main()

File: tests/test_data/ChatGPT/v1/Transcribed_JS/Conversor.js
class Conversor {
  constructor() {
    this.convertirTemperatura();
  }

  async convertirTemperatura() {
    const temperaturaCelsius = await this.ingresarTemperatura();
    const fahrenheit = (temperaturaCelsius * 9/5) + 32;
    console.log(`Equivalente en Fahrenheit: ${fahrenheit}`);
  }

  async ingresarTemperatura() {
    const respuesta = await prompt("Ingrese la temperatura en Celsius: ");
    return parseFloat(respuesta);
  }
}

const conversor = new Conversor();